dashboard/app/models/pd/teacher_application.rb

```diff
@@ -18,6 +18,33 @@
 #
 
 class Pd::TeacherApplication < ActiveRecord::Base
+  PROGRAM_DETAILS_BY_COURSE = {
+    'csd' => {
+      name: 'CS Discoveries',
+      url: 'https://code.org/educate/professional-learning/cs-discoveries',
+      approval_form_id: '1FAIpQLSdcR6oK-JZCtJ7LR92MmNsRheZjODu_Qb-MVc97jEgxyPk24A'
+    },
+    'csp' => {
+      name: 'CS Principles',
+      url: 'https://code.org/educate/professinoal-learning/cs-principles',
+      approval_form_id: '1FAIpQLScVReYg18EYXvOFN2mQkDpDFgoVqKVv0bWOSE1LFSY34kyEHQ'
+    }
+  }
+
+  REQUIRED_APPLICATION_FIELDS = %w[
```

Comment:
There are a lot more required fields (survey, grades planning to teach, etc). Do you intend to validate all?